src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,9 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ todos }) => (
+  <div className="PostList">
+    {todos.map(todo => (
+      <PostInfo key={todo.id} todo={todo} />
+    ))}
+  </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings

Reply:
Not fixed
<img width="771" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/98764641/f7afdf05-a755-4448-8d1a-0c44ca6eaa3e">